webapp/views.py

```diff
@@ -921,4 +922,4 @@ def marketo_submit():
     if return_url:
         return flask.redirect(return_url)
 
-    return flask.jsonify({"message": "Form submitted."})
```

Comment:
Was *every* marketo form previously a background-submit one? That's shocking!

Reply:
Yes, all of them

Reply:
:facepalm: